### PR TITLE
added additional support

### PR DESCRIPTION
**Pull request by riba2101**
### Reasons for making this change

added additional support #228
If this is related to existing tickets, include links to them as well.

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've checked the rendering of the Markdown text I've added
  - [ ] If I'm adding a new section, I've updated the Table of Content
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated docs if needed
* [x] **I'm adding a new feature**
  - [x] I've updated the playground with an example use of the feature

just use a scheme that contains the key

Schema:

```jsx
{
	"title": "A registration form",
	"description": "A simple form example.",
	"$schema": "http://json-schema.org/draft-04/schema#",
	"type": "object",
	"properties": {
		"name": {
			"type": "string"
		},
		"devMap": {
			"type": "object",
			"additionalProperties": {
				"type": "integer"
			}
		}
	}
}
```

formData:

```jsx
{
	"name": "test",
	"devMap": {
		"key": 2,
		"value": 12
	}
}
```

### CODE REVIEW

**Comment by jManji** [comment]
@riba2101 Does this support `uiShema`?
Also you've ticked that you have updated the playground, but I don't see in the code that you have...

**Comment by riba2101** [comment]
uiSchema support is somewhat basic, would be grate to extend it but given my "no time atm to play"...
added to playgorund(i've pasted the example here in the PR)

File: src/components/fields/ObjectField.js [hunk 1]
```diff
@@ -84,67 +81,124 @@ class ObjectField extends Component {
...
+                      id={`${idSchema.$id}__description`}
+                      description={schema.description}
+                      formContext={formContext}/> : null}
+              {Object.keys(this.state).map((name, index) => {
+                const childIdSchema = {"$id": `${idSchema.$id}__${name}`};
+                return (
+                      <SchemaField key={index}
+                                   name={name}
+                                   required={this.isRequired(name)}
+                                   schema={schema.additionalProperties}
```
Comment (glasserc) [inline src/components/fields/ObjectField.js]: I'm pretty sure this doesn't correctly support a schema that has both `properties` and `additionalProperties`.

File: src/utils.js [hunk 2]
```diff
@@ -129,6 +129,10 @@ function computeDefaults(schema, parentDefaults, definitions={}) {
   }
   // We need to recur for object schema inner default values.
   if (schema.type === "object") {
+    if (!schema.properties){
+      return defaults;
+    }
```
Comment (glasserc) [inline src/utils.js]: What is this change about?

**Comment by nhpace** [comment]
What is the status of this. Is it being actively worked on?

**Comment by riba2101** [comment]
Hey, sry but I'm currently seriously lacking time. Feel free to take it over

**Comment by epicfaace** [comment]
Closing this PR as additionalProperties support was added in #1021.

**Pull request closed without merging**